Sources/BraintreePayPalNativeCheckout/BTPayPalNativeOrderCreationClient.swift

```diff
@@ -62,6 +65,7 @@ class BTPayPalNativeOrderCreationClient {
             ) { json, response, error in
                 guard let hermesResponse = BTPayPalNativeHermesResponse(json: json), error == nil else {
                     let underlyingError = error ?? BTPayPalNativeError.invalidJSONResponse
+                    self.apiClient.sendAnalyticsEvent("ios.paypal-native.create-order.order-creation.failed")
```

Comment:
`.create-order.order-creation.failed` is a little confusing. Maybe `.create-order.hermes-request.failed`?